src/ebay-tri-state-checkbox/__tests__/index.stories.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState, useRef, ChangeEvent } from 'react'
+import { StoryFn, Meta } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { EbayLabel } from '../../ebay-field'
+import { EbayTriStateCheckbox } from '../index'
+
+const meta: Meta<typeof EbayTriStateCheckbox> = {
+    component: EbayTriStateCheckbox,
+    title: 'form input/ebay-tri-state-checkbox',
+    argTypes: {
+        checked: {
+            options: ["false", "mixed", "true"],
+            type: { category: "Options" },
+            description:
+                'Either "true", "false" or "mixed". Defaults to "false". Changes the checkbox state to the given one depdending on the checked state.',
+            table: {
+                defaultValue: {
+                    summary: "false",
+                },
+            },
+        },
+        skipMixed: {
+            type: "boolean",
+            control: { type: "boolean" },
+            description:
+                "If set, then will skip the mixed toggle when clicking on checkbox. Used if in some cases you want to toggle between all items selected or none.",
+        },
+        size: {
+            options: ["default", "large"],
+            type: { category: "Options" },
+            description:
+                'Either "large" or "default". Sets the checkbox icon. Default is regular. For mweb this should be set to large. (Note: The dimensions of the checkbox will not change, but only the icon)',
+            table: {
+                defaultValue: {
+                    summary: "default",
+                },
+            },
+        },
+        onChange: {
+            action: "on-change",
+            description: "Triggered on change",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value, checked }",
+                },
+            },
+        },
+        onFocus: {
+            action: "on-focus",
+            description: "Triggered on focus",
+            table: {
+                category: "Events",
+                defaultValue: {
+                    summary: "{ originalEvent, value }",
+                },
+            },
+        },
+    },
+}
+
+export default meta
+
+export const Default: StoryFn<typeof EbayTriStateCheckbox> = (args) => (
+    <>
+        <p style={{ alignItems : "center" }}>
+            <EbayTriStateCheckbox
+                {...args}
```

Comment:
since we are passing the `...args` we don't need to set any property as it is already handle by the `argTypes` definition.

Consider this for all the stories bellow also